Mathlib/CategoryTheory/ConcreteCategory/Basic.lean

```diff
@@ -153,45 +150,6 @@ theorem ConcreteCategory.mono_of_injective {X Y : C} (f : X ⟶ Y) (i : Function
   (forget C).mono_of_mono_map ((mono_iff_injective f).2 i)
 #align category_theory.concrete_category.mono_of_injective CategoryTheory.ConcreteCategory.mono_of_injective
```

Comment:
Can you also move this one?
I think `import Mathlib.CategoryTheory.Functor.EpiMono` won't be needed after this.

Reply:
And you can use `assert_not_exists` to guarantee the imports aren't changed for now, but if you've got a better solution down the road feel free to ignore.

Reply:
Done, thanks for the suggestions!